### PR TITLE
#7Task. yulia.miasnikova

### PR DESCRIPTION
**Pull request by YuliaSirius**
https://travis-ci.org/github/YuliaSirius/js-assignments/builds/710001490

### CODE REVIEW

**Comment by AlexMokin** [comment]
Pr must include changes only in files related to the given task
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by YuliaSirius** [comment]
Please tell me if I did it wrong. I removed unnecessary commits with the command git reset , and add necessary commit.

**Comment by YuliaSirius** [comment]
Travis CI link:
https://travis-ci.org/github/YuliaSirius/js-assignments/builds/710286204

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -145,15 +207,21 @@ function* mergeSortedSequences(source1, source2) {
...
-
 module.exports = {
-    get99BottlesOfBeer: get99BottlesOfBeer,
-    getFibonacciSequence: getFibonacciSequence,
-    depthTraversalTree: depthTraversalTree,
-    breadthTraversalTree: breadthTraversalTree,
-    mergeSortedSequences: mergeSortedSequences,
-    async               : async
+  get99BottlesOfBeer: get99BottlesOfBeer,
+  getFibonacciSequence: getFibonacciSequence,
```
Comment (AlexMokin) [inline task/07-yield-tasks.js]: do not change formatting of the original code and comments

**Comment by AlexMokin** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by YuliaSirius** [comment]
I create a new PR instead of this- #755.

**Pull request closed without merging**